content/realtime/presence.textile

```diff
@@ -899,6 +901,12 @@ h4. Parameters
 - <div lang="swift,objc">callback</div> := called with each matching "@presence message@":#presence-message when received on the channel
 - <div lang="csharp">handler</div> := called with each matching "@presence message@":#presence-message when received on the channel
 
+h4. Considerations
+
+* If the channel is @initialized@ (i.e. no attempt to attach has yet been made for this channel), then calling @subscribe@ will implicitly attach the channel. However, regardless of the implicit attach outcome, the <span lang="default">listener</span><span lang="ruby">block</span><span lang="csharp">handler</span> will still be registered
+* If @subscribe@ is called more than once with the same <span lang="default">listener</span><span lang="ruby">block</span><span lang="csharp">handler</span>, then duplicates will be registered. For example, if you @subscribe@ twice with the same <span lang="default">listener</span><span lang="ruby">block</span><span lang="csharp">handler</span> and a presence message is later received, the same <span lang="default">listener</span><span lang="ruby">block</span><span lang="csharp">handler</span> will be invoked twice
+* The registered <span lang="default">listener</span><span lang="ruby">block</span><span lang="csharp">handler</span> remains active on the presence channel regardless of the underlying channel state. For example, if you call @subscribe@ when the underlying channel is @attached@ and it later becomes @detached@ or even @failed@, when the channel is reattached and a presence message is received, the <span lang="default">listeners</span><span lang="ruby">blocks</span><span lang="csharp">handlers</span> originally registered will still be invoked. Listeners are only removed when calling "@unsubscribe@":#unsubscribe or when the underlying channel is @released@ using the <span lang="default">@Realtime.channels.release(name)@</span><span lang="csharp">@Realtime.Channels.Release(name)@</span> method
```

Comment:
> The js docs generally use realtime rather than Realtime for an instance of the client lib,

Well as far as I can tell, we haven't done either. We only use `realtime.*` in code example, never in documentation itself. I searched the docs now, cannot find one instance of that or channel.* etc either.

Unfortunately the issue here is in fact that we don't have docs for the Channels collection, see https://github.com/ably/docs/issues/251. I think I'm going to leave this as is until we address that problem and then we can use the pattern we have used in the docs such as "... if either @publish@ or @subscribe@ for the @Channel@ is called ..."